Shared/Samples/Generate geodatabase replica from feature service/README.metadata.json

```
@@ -0,0 +1,34 @@
+{
+    "category": "Edit and Manage Data",
+    "description": "Generate a local geodatabase replica from an online feature service.",
+    "ignore": false,
+    "images": [
+        "generate-geodatabase-replica-from-feature-service.png"
+    ],
+    "keywords": [
+        "disconnected",
+        "local geodatabase",
+        "offline",
+        "replica",
+        "sync",
+        "GenerateGeodatabaseJob",
+        "GenerateGeodatabaseParameters",
+        "Geodatabase",
+        "GeodatabaseSyncTask"
+    ],
+    "offline_data": [
+        "e4a398afe9a945f3b0f4dca1e4faccb5"
+    ],
+    "redirect_from": [],
+    "relevant_apis": [
+        "GenerateGeodatabaseJob",
+        "GenerateGeodatabaseParameters",
+        "Geodatabase",
+        "GeodatabaseSyncTask"
+    ],
+    "snippets": [
+        "GenerateGeodatabaseReplicaFromFeatureServiceView.Model.swift",
+        "GenerateGeodatabaseReplicaFromFeatureServiceView.swift"
```

```suggestion
        "GenerateGeodatabaseReplicaFromFeatureServiceView.swift",
        "GenerateGeodatabaseReplicaFromFeatureServiceView.Model.swift"
```
The main view's filename needs to be the first in the list, so the app will pick it up when the sample is built.